roles/StackStorm.rabbitmq/defaults/main.yml

```
@@ -5,3 +5,5 @@ rabbitmq_plugins: []
 #  - rabbitmq_management
 # Set to "present" to install latest version, or specify specific version
 rabbitmq_version: "present"
+# Use version or wildcard
+erlang_el8_version: "24*"
```

```suggestion
erlang_version: "24*"
```
It's not el8 specific, right?

---

> It's not el8 specific, right?

Only EL8 failed last night. Looking at the packagecloud rabbit repo. EL8 has 25*, 24*, 23*. EL7 only has 23*.
So problem is EL8 specific at moment, and likely to not update EL7 as they never updated it for 24.

But you are right I at least need to do something as the file changed affects EL7 and EL8, but I'll have to have default versions for the two :-(

---

In that case, you can add an extra `if` condition on the task that only uses the pinned version for el8